steppy/base.py

```diff
@@ -367,18 +376,24 @@ def transform(self, data):
                 step_inputs = self._adapt(step_inputs)
             else:
                 step_inputs = self._unpack(step_inputs)
-            step_output_data = self._cached_transform(step_inputs)
+            step_output_data = self._transform_operation(step_inputs)
         return step_output_data
 
-    def clean_cache(self):
-        """Removes everything from the directory ``<experiment_directory>/cache``.
+    def clean_cache_step(self):
+        """Clean cache for current step.
+        """
+        logger.info('Step {}, cleaning cache'.format(self.name))
+        self.output = None
+        gc.collect()
+
+    def clean_cache_pipeline(self):
```

Comment:
@kamil-kaczmarek I think we are using upstream when referring to pipeline (like in save_visualization_graph_upstream).

Reply:
corrected